File: EarthSentry.Frontend/earthsentry-app/src/interfaces/PostCommentDto.ts
export interface PostCommentProps extends PostCommentDto {
  onDelete?: (commentId: number) => void;
}

export interface PostCommentDto {
    commentId: number;
    userImageUrl: string;
    username: string;
    content: string;
    createAt: Date;
    isFromOwner: boolean;
};

File: EarthSentry.Contracts/Contracts/Posts/PostCommentDto.cs
namespace EarthSentry.Contracts.Contracts.Posts
{
    public class PostCommentDto
    {
        public int CommentId { get; set; }
        public string UserImageUrl { get; set; }
        public string Username { get; set; }
        public string Content { get; set; }
        public DateTime CreateAt { get; set; }
        public bool IsFromOwner { get; set; }
    }
}
